test/components/views/beacon/RoomCallBanner-test.tsx

```diff
@@ -0,0 +1,135 @@
+/*
+Copyright 2022 The Matrix.org Foundation C.I.C.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+import React from "react";
+import { act } from "react-dom/test-utils";
+import {
+    Room,
+    PendingEventOrdering,
+    MatrixClient,
+    RoomMember,
+    RoomStateEvent,
+} from "matrix-js-sdk/src/matrix";
+import { ClientWidgetApi, Widget } from "matrix-widget-api";
+import {
+    cleanup,
+    render,
+    screen,
+} from "@testing-library/react";
+import { mocked, Mocked } from "jest-mock";
+
+import {
+    mkRoomMember,
+    MockedCall,
+    setupAsyncStoreWithClient,
+    stubClient,
+    useMockedCalls,
+} from "../../../test-utils";
+import RoomCallBanner from "../../../../src/components/views/beacon/RoomCallBanner";
+import { CallStore } from "../../../../src/stores/CallStore";
+import { WidgetMessagingStore } from "../../../../src/stores/widgets/WidgetMessagingStore";
+import { MatrixClientPeg } from "../../../../src/MatrixClientPeg";
+import { RoomViewStore } from "../../../../src/stores/RoomViewStore";
+
+describe("<RoomCallBanner />", () => {
```

Comment:
Would be nice to test clicking buttons though due to time, I don't think it is necessary. I would suggest a TODO

Reply:
added the TODO comment